oc_ldap_user_locker/locker.py

```diff
@@ -160,18 +161,46 @@ def _check_user_conf(self, user_rec, conf):
         _matched_attributes = 0
         for _attrib in conf['condition_attributes'].keys():
             logging.debug("Comparing attribute: '%s'" % _attrib)
-            _vals_from_rec = user_rec.get_attribute(_attrib)
             _match_conf = conf['condition_attributes'][_attrib]
+            
+            if "." not in _attrib:
```

Comment:
Bad place to do so.
With such an implementation we are limiting ourselves with two-level comparison. That is `a.b.c.` will not work.
Please move this entire logic to `_compare_attribute`